### ISSUE TITLE
Chp. 3 - Misleading description of Moment of Inertia

### ISSUE THREAD

**Issue by dipamsen** [nature-of-code/noc-book-2#502]
> **Torque** is a measure of the rotational force acting on an object. In the case of a pendulum, torque is proportional to both the mass and the length of the arm ($M \times r$). The **moment of inertia** of a pendulum is a measure of how difficult it is to rotate the pendulum around the pivot point. It’s proportional to the square of the length of the arm ($r^2$).  
> 
> By dividing the torque by the moment of inertia ($Mr / r^2 ⇒ M / r$), I can calculate the angular acceleration of the pendulum more accurately. In fact, I can continue to ignore mass, as it has no actual effect on the acceleration: it scales the force of gravity, which contributes to the force of the pendulum ($F_p)$, but it also divides the force of the pendulum ($A = F_p / M$) to calculate the acceleration.

Here, torque is described as proportional to the mass _of the bob_ (this may be explicitly mentioned) and length of the arm. Similarly, the moment of inertia is also proportional to the **mass of bob**, and square of the length of the arm ($I = M r^2$) ([source](https://en.wikipedia.org/wiki/Moment_of_inertia)). The mass proportionality of $I$ has been ignored due to which the resulting angular acceleration seems to be mass-dependent, and it is later implied in the text that angular acceleration is independent of mass.

This can just be seen from the mathematical derivation, as $\text{angular acceleration } \propto \tau / I = Mr / Mr^2 = 1/r$, so the correction term in the derived expression for angular acceleration is $1/r$. (independent of mass)

tldr - corrections:
1. "torque is proportional to both the mass **of the bob** and the ..."
2. [moment of inertia] "It’s proportional to **both the mass of the bob, and** the square of the length of the arm **($M r^2$)**." 
3. Calculation of angular acceleration - $Mr / Mr^2 = 1/r$

**Comment by shiffman** [comment]
Thank you @dipamsen! Here it is, let me know what you think! cc @wellcaffeinated @nheidelberger 

This is a good time for a reminder that the context here is creative coding and not pure physics. Yes, the acceleration due to gravity on Earth is 9.8 meters per second squared. But this number isn’t relevant here in the world of pixels. Instead, I’ll use an arbitrary constant (called `gravity`) as a variable that scales the acceleration. (Incidentally, angular acceleration is usually written as $\alpha$ so as to distinguish it from linear acceleration $A$.)

$$
\alpha = \text{gravity} \times \sin(\theta) 
$$

Before I put everything together, there’s another detail I neglected to mention. Or really, lots of little details. Think about the pendulum arm for a moment. Is it a metal rod? A string? A rubber band? How is it attached to the pivot point? How long is it? What’s its mass? Is it a windy day? There are a lot of questions that I could continue to ask that would affect the simulation. I choose to live, however, in a fantasy world, one where the pendulum’s arm is some idealized rod that never bends and where the mass of the bob is concentrated in a single, infinitesimally small point. 

Even though I prefer not to worry myself with all of these questions, there’s a critical missing piece here related to the calculation of angular acceleration. To keep things simple, in the derivation of the pendulum’s angular acceleration, I assumed that the length of the pendulum’s arm is 1. In reality, however, the length of the pendulum’s arm affects the acceleration of the pendulum due to the concepts of torque and moment of inertia. 

**Torque** (**or** $\tau$) is a measure of the rotational force acting on an object. In the case of a pendulum, torque is proportional to both the mass of the bob and the length of the arm ($M \times r$). The **moment of inertia** (or ****$I$) of a pendulum is a measure of how difficult it is to rotate the pendulum around the pivot point. It’s proportional to the the mass of the bob, and the *square* of the length of the arm ($Mr^2$).  

Remember Newton’s second law? $F=M \times A$

Well, it has a rotational counterpart. $\tau = I \times \alpha$

By rearranging  the equation to solve for the angular acceleration $\alpha$, I get $\alpha = \tau/I$. Simplifying further, this becomes $Mr/Mr^2$ or $1/r$.

The angular acceleration doesn't depend on the pendulum's mass! This is just like Galileo's Leaning Tower of Pisa experiment and linear acceleration. Different objects, regardless of their mass, fell at the same rate. And here, once again, the mass of a bob doesn't influence its acceleration—only the length of its arm does. Thus, the final formula becomes:

$$
\alpha = \frac{\text{gravity} \times \sin(\theta)}{r} 
$$

**Comment by shiffman** [comment]
Closing this issue!
